python/cudf/dataframe/columnops.py

```diff
@@ -263,12 +269,12 @@ def as_column(arbitrary, nan_as_null=True, dtype=None):
             arbitrary.data,
             arbitrary.dtype,
             mask=arbitrary.mask,
-            categories=categories
+            categories=categories,
+            name=name
```

Comment:
You don't need to pass the name here if you're setting it below.

Reply:
fixed it.